package.json

```diff
@@ -53,19 +53,19 @@
 		"eslint-plugin-import": "^2.28.0",
 		"eslint-plugin-jest": "^27.2.3",
 		"eslint-plugin-jsdoc": "^46.4.6",
-		"eslint-plugin-prettier": "^4.2.1",
+		"eslint-plugin-prettier": "^5.0.0",
 		"express-basic-auth": "^1.2.1",
 		"husky": "^8.0.3",
```

Comment:
do we need to update the husky config?

Reply:
In the file `.husky/pre-commit` is the command `npm run lint:staged`. I adapted the command for `lint:staged` in the `package.json`. I think that should be enough.